### PR TITLE
Add blank work template

### LINKED ISSUE #1419: On a work show page analytics blows up
Missing Template: stats/work

### PR DESCRIPTION
**Pull request by njaffer**
Closes #1419. This would show a blank analytics page. A new ticket #1455 is created to do more work on analytics

### CODE REVIEW

**Comment by jcoyne** [comment]
@njaffer can you please add a test to app/controllers/stats_controller_spec.rb for `get :work, id: '123'` that shows this change worked?

**Comment by mjgiarlo** [comment]
@jcoyne She's going to add a test that makes sure a `GET` to that path returns a successful HTTP response (as discussed on today's standup).

File: spec/controllers/stats_controller_spec.rb [hunk 1]
```diff
@@ -72,4 +72,14 @@
...
     end
   end
+
+  describe 'work' do
+    routes { Sufia::Engine.routes }
+
+    it 'renders the stats view' do
+      get :work, id: '123'
+      expect(response).to be_success
+      #          expect(response).to render_template('stats/work')
```
Comment (awead) [inline spec/controllers/stats_controller_spec.rb]: did you want to leave this in? If not, you can just delete the line.

Comment (njaffer) [inline spec/controllers/stats_controller_spec.rb]: @awead: I can remove the line, the test is still failing though

**Pull request merged**